app/helpers/ExerciseConfig/Compilation/PipelinesMerger.php

```diff
@@ -142,7 +142,10 @@ private function mergeTrees(MergeTree $first, MergeTree $second): MergeTree {
     }
 
     // transform remaining output from first tree into classic array
-    $newFirstOutput = array_values($outVars);
+    $newFirstOutput = [];
+    foreach ($outVars as $outVar) {
```

Comment:
If only `array_map` was not the verbose beast it is in PHP...

Reply:
Well sorry, I am more like the classical foreach guy :-) but can change it if you want.

Reply:
That was meant to be a criticism of your code, just a little rant. Foreach is OK.